### DISCUSSION TITLE
Impact of Terraform license change

### DISCUSSION THREAD

**Discussion by pratheekrebala** [coder/coder#9241] [Announcements]
Hey Everyone,

I just saw the news regarding the [changes to Terraform's licensing <sup>[1]</sup>](https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license) and was curious what impact this would have on Coder and the current roadmap.

Thank you again for all your work on this amazing product!

<sup>[1]</sup> https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license

**Comment by kylecarbs** [comment]
Hey @pratheekrebala!

We aren't directly competitive with HashiCorp, so we don't expect any immediate impact. We'll be checking with them to be confident of the long-term implications.

The release of Terraform currently used in the product is MPL, and when that changes, we'll update this issue.

**Comment by matifali** [comment]
Here is Coder's [blog post](https://coder.com/blog/hashicorp-license) on this.

**Comment by pratheekrebala** [comment]
Thank you for clarifying this @matifali!
